docs/README.MD

```diff
@@ -40,7 +40,7 @@ DogeHouse makes it easy to connect with friends and people over voice and chat m
 - Delete message
   You can only delete messages if:
   - You wrote it.
-  - You are a mod
```

Comment:
I think it should either remain mod or all instances of mod should be switched to moderator for consistency.